public/admin/index.php

```diff
@@ -50,7 +50,7 @@
 }
 
 if (in_array(22, $droits)) {
-    echo "<li><a href='index.php?page=planning/postes_cfg/index.php'>Les tableaux</a></li>\n";
+    echo "<li><a href='/framework'>Les tableaux</a></li>\n";
```

Comment:
please, use asset

Reply:
As it not a twig template, shouldn't I use {$config['URL']} instead ?